### DISCUSSION TITLE
Distributed setting: Multiple nodes accessing network-shared mapped drive

### DISCUSSION THREAD

**Discussion by eladn** [tox-dev/filelock#189] [Q&A] [answered]
Is the file-locking mechanism expected to work across nodes in a cluster accessing the same mapped drive shared over the network, or is the locking performed only on the OS-level?

**Comment by gaborbernat** [comment, marked as answer]
Hard to answer that. We use OS primitives to acquire the lock, so it depends if your OS supports these primitives on the mapped drives. If not the bug is in the OS.

**Reply by eladn** [reply]
Thanks. Can you point out which OS primitives does FileLock use for UNIX?
